### DISCUSSION TITLE
支付宝转账失败 无法读取支付宝返回的错误异常

### DISCUSSION THREAD

**Discussion by ChenjunGitA** [yansongda/pay#581] [Q&A] [answered]
在进行支付宝转账的时候 支付宝日志返回：收款账号不存在或户名有误，建议核实账号和户名是否准确
我想读取这句话，但是一直无法捕获。只能报如下错误：
异常原因：Yansongda\Pay\Exceptions\GatewayException: ERROR_GATEWAY: Get Alipay API Error:Business Failed - PAYEE_NOT_EXIST in /data/wwwroot/test.chxchina.com/vendor/yansongda/pay/src/Gateways/Alipay/Support.php:408
Stack trace:
#0 /data/wwwroot/test.chxchina.com/vendor/yansongda/pay/src/Gateways/Alipay/Support.php(147): Yansongda\Pay\Gateways\Alipay\Support::processingApiResult()
有什么好的方法可以获取到返回的sub_msg

**Comment by yansongda** [comment, marked as answer]
`raw` 里能取吧？

PS：建议升级到 V3

**Reply by ChenjunGitA** [reply]
![image](https://user-images.githubusercontent.com/45305258/156872928-a42b344a-89ac-48a1-99a9-8b6586fa900f.png)

**Reply by ChenjunGitA** [reply]
这个是我的代码 你说的raw是怎样使用呢 谢谢你的回复

**Reply by yansongda** [reply]
`$e->raw`

**Reply by ChenjunGitA** [reply]
![image](https://user-images.githubusercontent.com/45305258/156873587-46637f85-a55a-45e6-9fbf-990adea30bee.png)
修改了之后，说找不到之后方法
![image](https://user-images.githubusercontent.com/45305258/156873600-cb01d3fa-42a3-4257-9388-ff6382737976.png)

**Reply by ChenjunGitA** [reply]
您好 麻烦您帮我看看 万分感激 有偿的也行 谢谢
